OTAcceleratorCore/OTMultiPartyCommunicator.m

```diff
@@ -222,11 +213,23 @@ - (NSError *)connect {
 }
 
 - (NSError *)disconnect {
+    MultiPartyLoggingWrapper *loggingWrapper = [MultiPartyLoggingWrapper sharedInstance];
+    
+    [loggingWrapper.logger logEventAction:KLogActionDisconnect
+                                variation:KLogVariationAttempt
+                               completion:nil];
     
     // need to explicitly unpublish and unsubscriber if the communicator is the only accelerator to dismiss from the common session
     // when there are multiple accelerator packs, the common session will not call the disconnect method until the last delegate object is removed
     if (self.publisher) {
-        
+        [loggingWrapper.logger logEventAction:KLogActionStopPublishing
+                                    variation:KLogVariationAttempt
+                                   completion:nil];
+        if (self.screenSharingView) {
+            [loggingWrapper.logger logEventAction:KLogActionStopScreensharing
+                                        variation:KLogVariationAttempt
+                                       completion:nil];
+        }
```

Comment:
After review it, I agree. Maybe, it could be confused, so, we are going to log:
- start/stop publishing media (camera stream)
- start/stop screensharing

I will add the modification to android core as well. thanks!!